README.md

```diff
@@ -314,6 +288,137 @@ folders. To test if everything works open the demo.html file.
+has a scrollable-container (another component) in its template and all options (list items) of the dropdown, which are passed through slots as they are dynamic go into the scrollable container:
+
+~~~~{.html}
+<div class="dropdown">
+    <div class="dropdown-header">
+        <div class="selected">Select an option</div>
+        <div class="custom-select-arrow"></div>
+    </div>
+    <div class="options-container hidden">
+        <gameface-scrollable-container class="scrollable-container-component">
+            <div slot="scrollable-content" data-name="scrollable-content">
+                <div class="options">
+                    <!--all options go here-->
+                    <component-slot data-name="option"></component-slot>
+                </div>
+            </div>
+        </gameface-scrollable-container>
+    </div>
+</div>
+~~~~
+
+The usage of the component looks a lot simpler:
+
+~~~~{.html}
+<gameface-dropdown class="gameface-dropdown-component">
+    <dropdown-option slot="option">Cat1</dropdown-option>
+    <dropdown-option slot="option" disabled>Cat1</dropdown-option>
+    <dropdown-option slot="option" disabled>Cat1</dropdown-option>
+    <dropdown-option slot="option">Cat3</dropdown-option>
+</gameface-dropdown>
+~~~~
+
+And when the component is rendered and all elements are slotted the final markup will look like this:
+
+~~~~{.html}
+<div class="dropdown">
+    <div class="dropdown-header">
+        <div class="selected">Select an option</div>
+        <div class="custom-select-arrow"></div>
+    </div>
+    <div class="options-container hidden">
+        <gameface-scrollable-container class="scrollable-container-component">
+            <div slot="scrollable-content" data-name="scrollable-content">
+                <div class="options">
+                    <dropdown-option slot="option">Cat1</dropdown-option>
+                    <dropdown-option slot="option" disabled>Cat1</dropdown-option>
+                    <dropdown-option slot="option" disabled>Cat1</dropdown-option>
+                    <dropdown-option slot="option">Cat3</dropdown-option>
+                </div>
+            </div>
+        </gameface-scrollable-container>
+    </div>
+</div>
+~~~~
+
 # Testing
 
 All tests are located in tools/tests. For more information on how to create and run a test refer to the [documentation](https://github.com/CoherentLabs/GameUIComponents/blob/master/tools/tests/README.md).
+
+
+# Publishing components to npm
+
+Before you publish make sure:
+
+1. The component builds - the umd and cjs packages are successfully generated.
+2. All tests pass.
+3. There is a README.md with useful information about how to use the component.
+4. The name of the component is correct (prefixed with coherent-gameface-).
+5. The demo is bundled.
+6. There is a public documentation commited in the docs folder.
+

```

Comment:
Lets add a point that when we publish components we should add other members of the team as contributors in npm so that when necessary everyone can update the packages

Reply:
I had no idea this was mandatory. Isn't it too much work. What if a project has 1K+ contributors? 
I don't understand where others should be added as contributors? In package.json, or you need to somehow give them permissions to publish through your npm account? Doesn't this mean that you need to know their npm usernames? I haven't published a component yet, so I'm familiar with the overall procedure, don't know the details. Maybe @alien1976 can also help?

Reply:
Added.